include/NovelRT.Interop/Maths/NrtQuadTree.h

```diff
@@ -1,44 +1,39 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#ifndef NOVELRT_MATHS_QUADTREE_C_H
-#define NOVELRT_MATHS_QUADTREE_C_H
+#ifndef NOVELRT_INTEROP_MATHS_QUADTREE_H
+#define NOVELRT_INTEROP_MATHS_QUADTREE_H
 
-#include "NrtGeoBounds.h"
-#include "NrtQuadTreePoint.h"
-#include <stddef.h>
```

Comment:
All these missing includes are worrying me. I explicitly added them so that if you had access to the handle, you'd have access to the relevant functionality associated with that handle.

Reply:
The C files will primarily be used by interop bindings, not by actual C code, in which case it is more beneficial (in my experience) to have the typedefs centrally managed and defined.

As mentioned above, many many native libraries centralize their opaque types and definitions. Part of this is because they are often needed by many things below that point, but also because it allows you to easily see all the types at once: https://github.com/KhronosGroup/Vulkan-Headers/blob/master/include/vulkan/vulkan_core.h#L57. See also: `X11/X.h`, most of the Windows SDK (particularly DirectX and friends), `gl.h`, many of our own dependencies, etc.